rmf_door_msgs/msg/DoorMode.msg

```diff
@@ -0,0 +1,10 @@
+# The DoorMode message captures the "mode" of an automatic door controller.
+# Most door controllers default to running in "closed" mode, and transition
+# through some sort of "moving" mode until reaching the "open" mode.
+
+uint32 mode
```

Comment:
```suggestion
uint32 value
```

This way a user would type `door_state_msg.current_mode.value` instead of `door_state_msg.current_mode.mode`.